lib/policies/jwt/index.js

```diff
@@ -0,0 +1,32 @@
+const extractors = require('./extractors');
+
+module.exports = {
+  policy: require('./jwt'),
+  schema: {
+    type: 'object',
+    properties: {
+      secret: {
```

Comment:
I think `secret` and `secretFile` should be `secretOrPublicKey` and `secretOrPublicKeyFile`, right?  It can be a public key depending on the algorithm (HMAC, RSA, ECDSA).

Reply:
That's correct. You can technically put the public key there as a string.
Can we try to find a more generic name that would encompass both cases?

`secretOrPublicKey` isn't very appealing to me. I'd rather try to find a generic name that would make users understand this parameter means this or that, such as `signature`, something like that.

Reply:
Tyk is calling this field `jwt_source`.
Kong is using two different fields: `secret` and `rsa_public_key`

Reply:
Done in 6c77e3c